assets/js/blocks/product-query/inspector-controls.tsx

```diff
@@ -221,6 +223,11 @@ const ProductQueryControls = ( props: ProductQueryBlock ) => {
 	return (
 		<>
 			<InspectorControls>
+				{ MANUAL_REPLACE_PRODUCTS_WITH_PRODUCT_COLLECTION ? (
+					<UpgradeNotice
+						upgradeBlock={ replaceProductsWithProductCollection }
+					/>
+				) : null }
```

Comment:
Perhaps this can be simplified by removing the ternary operator?

```suggestion
				{ MANUAL_REPLACE_PRODUCTS_WITH_PRODUCT_COLLECTION && (
					<UpgradeNotice
						upgradeBlock={ replaceProductsWithProductCollection }
					/>
				)}
```

Reply:
Sure! Addressed in this [commit](https://github.com/woocommerce/woocommerce-blocks/pull/10268/commits/9039c6d10f891279603cf278e91f2c0f0df47f37).